.github/workflows/ci.yml

```diff
@@ -120,6 +120,22 @@ jobs:
           bazel test //tensorboard/summary/writer:event_file_writer_s3_test &&
           bazel test //tensorboard/compat/tensorflow_stub:gfile_fsspec_test &&
           bazel test //tensorboard/summary/writer:event_file_writer_fsspec_test
+      - name: 'Bazel: build Pip package (with TensorFlow only)'
+        if: matrix.tf_version_id == 'tf'
+        run: |
+          version="$(grep -Po "(?<=VERSION = ['\"]).*?(?=['\"])" tensorboard/version.py)"
```

Comment:
Optional, but we might consider factoring out the `grep` and `sed` commands here (the first 5 lines) into a separate shell script (e.g. like `tensorboard/tools/update_version.sh`) so that they're a bit easier to test and invoke manually?

In particular, it might be nice to be able to extend these commands so that we could have this same build step work for doing actual releases (not just nightly), in which case we'd need to make the logic a bit more complicated, and that would probably be best to do in a separate script vs inline here.  However, fine to leave as-is for now if you prefer.

Reply:
Good point! Done.